Cargo.lock

```diff
@@ -194,24 +194,12 @@ dependencies = [
  "hmac 0.11.0",
  "k256",
```

Comment:
It is still here, you may need `default-features = false` on `bip32`.

Reply:
Yep: https://github.com/iqlusioninc/crates/blob/main/bip32/Cargo.toml#L41

Reply:
But now it's gone :) 
[`32ba599` (#193)](https://github.com/comit-network/waves/pull/193/commits/32ba59915b96a2fb70c4c4080cd6b55e7a7164ec)

Reply:
-11 dependencies :partying_face: